src/components/appNavigation/TabsView.js

```diff
@@ -120,33 +120,79 @@ const TabsView = React.memo((props: TabViewProps) => {
     }
   }
 
-  return (
-    <Appbar.Header dark>
-      {config.isEToro && (
-        <TouchableOpacity testID="rewards_tab" onPress={goToRewards} style={styles.rewardsStyle}>
-          <Icon name="rewards" size={36} color="white" />
-        </TouchableOpacity>
-      )}
-      <Appbar.Content />
+  // default header buttons
```

Comment:
too complicated. Could be simplified

1. move market & etoro stuff to the separate components and wrap onto fragments:

```
const MarketButtons = () => (<>
<Appbar.Content key="header-button-3" />,
       <TouchableOpacity
         key="header-button-4"
         testID="goodmarket_tab"
         onPress={goToMarketplace}
         style={[styles.marketIconBackground, styles.marginRight10]}
       >
         <Icon name="goodmarket" size={36} color="white" />
       </TouchableOpacity>
</>);

const EToroButtons = () => (<>
  <TouchableOpacity key="header-button-2" testID="rewards_tab" onPress={goToRewards} style={styles.marginLeft10}>
         <Icon name="rewards" size={36} color="white" />
       </TouchableOpacity>,
       <Appbar.Content key="header-button-3" />,
       <TouchableOpacity onPress={goToRewards}>
         <Icon name="invite2" size={36} color="white" testID="invite_tab" />
       </TouchableOpacity>,
</>);
```

2. just add them in jsx using && expressions:

```
return (
  <Appbar.Header dark>
    {config.isEToro && (<EToroButtons />)}
    {config.market && (<MarketButtons />)}
    // support buttons
    <Appbar.Content key="header-button-6" />,
    <TouchableOpacity key="header-button-6" onPress={goToSupport} style={styles.feedback}>
      <Icon name="support2" size={36} color="white" testID="support_tab" />
    </TouchableOpacity>
    // default header buttons
    <Appbar.Content key="header-button-0" />,
    <TouchableOpacity onPress={toggleMenu} key="header-button-1">
      <Icon name="settings" size={20} color="white" style={styles.marginRight10} testID="burger_button" />
    </TouchableOpacity>
  </Appbar.Header>
)
```

Reply:
@serdiukov-o-nordwhale 
I made changes regarding your suggestions - see new commit.